src/Illuminate/Routing/Route.php

```diff
@@ -195,7 +195,7 @@ protected function runCallable()
         $callable = $this->action['uses'];
 
         return $callable(...array_values($this->resolveMethodDependencies(
-            $this->parametersWithoutNulls(), new ReflectionFunction($this->action['uses'])
+            $this->parametersWithoutNulls(), new ReflectionFunction($callable)
```

Comment:
I`d like to say, that will be better to just remove a ` temporary variable `, since it is not used now.
What are you think about this?